### PR TITLE
feat: improve log

### PR DESCRIPTION
**Pull request by yinzhicheng66**
### Describe what this PR does / why we need it
application test clean

### Describe how you did it
add app.stop recive chan

### Special notes for reviews

### CODE REVIEW

File: pkg/pkg.go [hunk 1]
```diff
@@ -134,6 +138,10 @@ func HostName() string {
 	return hostName
 }
 
+func UUID() string {
```
Comment (hnlq715) [inline pkg/pkg.go:141]: 这个id可以去掉

Comment (hnlq715) [reply pkg/pkg.go:141]: 和appInstance 是一个意思

**Review by hnlq715 — CHANGES_REQUESTED** [review]
去掉UUID，用EnvAppInstance

File: pkg/pkg.go (hunk 1 above)
Comment (yinzhicheng66) [reply pkg/pkg.go]: UUID has been replaced by appInstance.
thx

**Review by hnlq715 — APPROVED** [review]
LGTM~

**Pull request merged**